lms/templates/darklang/preview_lang.html

```diff
@@ -11,5 +11,5 @@
 <%block name="bodyclass">pattern-library</%block>
 
 <%block name="content">
-    <%include file="/darklang/preview_lang_include.html" />
+    <%include file="/dark_lang/preview_lang_include.html" />
```

Comment:
I moved this template into the Django app as part of this PR. You can see that it is working on the sandbox.